### PR TITLE
VcNarrator: Add 'Narrate Self' option

### PR DESCRIPTION
**Pull request by iAmSpace**
if anyone wishes to hear their own name on join this should work

### CODE REVIEW

File: src/plugins/vcNarrator.tsx [hunk 1]
```diff
@@ -165,7 +165,7 @@ export default definePlugin({
                 if (!type) continue;
 
                 const template = Settings.plugins.VcNarrator[type + "Message"];
-                const user = isMe ? "" : UserStore.getUser(userId).username;
+                const user = Settings.plugins.VcNarrator["narrateSelf"] && isMe ? "" : UserStore.getUser(userId).username;
```
Comment (Vendicated) [inline src/plugins/vcNarrator.tsx:168]: narrateSelf is kind of a misleading name and this if should be inverted
```suggestion
                const user = isMe && !Settings.plugins.VcNarrator.sayOwnName ? "" : UserStore.getUser(userId).username;
```

**Comment by iAmSpace** [comment]
> narrateSelf is kind of a misleading name and this if should be inverted
that is my mistake sorry :)

File: src/plugins/vcNarrator.tsx [hunk 2]
```diff
@@ -230,6 +230,11 @@ export default definePlugin({
                 markers: [0.1, 0.5, 1, 2, 5, 10],
                 stickToMarkers: false
             },
+            narrateSelf: {
+                description: "Narrate Self",
```
Comment (Vendicated) [inline src/plugins/vcNarrator.tsx:234]: did you forget to update the setting

Comment (iAmSpace) [reply src/plugins/vcNarrator.tsx:234]: uhhh yes i am half asleep

**Review by Vendicated — APPROVED** [review]

**Pull request merged**